lib/net.js

```diff
@@ -1647,6 +1647,21 @@ function onconnection(err, clientHandle) {
   }
 
   if (self.maxConnections && self._connections >= self.maxConnections) {
+    const data = {};
+    if (clientHandle.getsockname) {
+      const localInfo = {};
+      clientHandle.getsockname(localInfo);
+      data.localAddress = localInfo.address;
+      data.localPort = localInfo.port;
+    }
+    if (clientHandle.getpeername) {
+      const remoteInfo = {};
+      clientHandle.getpeername(remoteInfo);
+      data.remoteAddress = remoteInfo.address;
+      data.remotePort = remoteInfo.port;
+      data.remoteFamily = remoteInfo.family;
+    }
+    self.emit('drop', data);
```

Comment:
For IPC connections `data` is an empty object. Is this wanted?

Reply:
> For IPC connections `data` is an empty object. Is this wanted?

Yes, i think return the `pipename` is not useful ?

Reply:
For IPC connections, return `undefined`. Thanks !